common/static/coffee/src/discussion/views/new_post_view.coffee

```diff
@@ -87,7 +87,6 @@ if Backbone?
               url: url
               type: "POST"
               dataType: 'json'
-              async: false # TODO when the rest of the stuff below is made to work properly..
```

Comment:
It looks like Arjun Singh added this on September 1st, 2012, and he included the TODO even back then. It appears that it was a transitional thing that was never revisited. I can see why it scares you, but we don't use synchronous AJAX anywhere else that I can think of. To me, blocking the browser waiting for the response seems worse.

If we don't make this change, we need to find a way to get all the Jasmine tests to work with the synchronous call. The way ``AjaxHelpers`` works, we invoke an action and then afterwards we declare the response to be given. This flow doesn't work with synchronous calls, so we need to instead declare what the result is going to be before we invoke the action.

Interestingly, synchronous AJAX has been deprecated since JQuery 1.8, so removing it seems necessary for us to upgrade:

http://stackoverflow.com/questions/11448011/jquery-ajax-methods-async-option-deprecated-what-now